src/cart/service/cart.service.ts

```diff
@@ -20,8 +21,9 @@ export class CartService {
     private readonly orderService: OrdersService,
   ) {}
 
-  async getItems(userId: number): Promise<CartItemsDto[]> {
+  async getItems(userId: number, pagination): Promise<PaginationCartItemDto> {
```

Comment:
I'd recommend you introduce a dto for pagination parameter here. 
It should be compatible with the GraphQL input and the REST dto.